### PR TITLE
Adds Firestarter bundle

### PR DESCRIPTION
**Pull request by LanCartwright**
A specialist, pyro kit from New Russian stockpiles.

This bundle contains appropriate paraphanalia, a pistol and ammo for problems, a fireproof hardsuit, a minibomb, and the star of the show; the backpack chemspray with it's massive  2000u multicylinder chemical storage system, filled to the brim with everything you need to ruin the days of any carbon lifeforms.

### CODE REVIEW

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 1]
```diff
@@ -488,3 +488,20 @@
 	new /obj/item/ammo_box/foambox/riot(src)
 	new /obj/item/weapon/grenade/chem_grenade/bioterrorfoam(src)
 	return
+
+/obj/item/weapon/storage/backpack/dufflebag/syndie/firestarter
+	desc = "A large dufflebag containing New Russian pyro backpack sprayer, a pistol, a pipebomb, fireproof hardsuit, ammo, and other equipment."
+
+/obj/item/weapon/storage/backpack/dufflebag/syndie/firestarter/New()
+	..()
+	contents = list()
```
Comment (duncathan) [inline code/game/objects/items/weapons/storage/backpack.dm]: I don't think this line is necessary?

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 2]
```diff
@@ -488,3 +488,20 @@
...
+	new /obj/item/clothing/under/syndicate/soviet(src)
+	new /obj/item/weapon/watertank/janitor/operator(src)
+	new /obj/item/clothing/suit/space/hardsuit/syndi/elite(src)
+	new /obj/item/weapon/gun/projectile/automatic/pistol/APS(src)
+	new /obj/item/ammo_box/magazine/pistolm9mm(src)
+	new /obj/item/ammo_box/magazine/pistolm9mm(src)
+	new /obj/item/weapon/reagent_containers/food/drinks/bottle/vodka/badminka(src)
+	new /obj/item/weapon/reagent_containers/syringe/stimulants(src)
+	new /obj/item/weapon/grenade/syndieminibomb(src)
+	return
```
Comment (duncathan) [inline code/game/objects/items/weapons/storage/backpack.dm]: Redundant return

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 3]
```diff
@@ -488,3 +488,20 @@
...
+
+/obj/item/weapon/storage/backpack/dufflebag/syndie/firestarter
+	desc = "A large dufflebag containing New Russian pyro backpack sprayer, a pistol, a pipebomb, fireproof hardsuit, ammo, and other equipment."
+
+/obj/item/weapon/storage/backpack/dufflebag/syndie/firestarter/New()
+	..()
+	contents = list()
+	new /obj/item/clothing/under/syndicate/soviet(src)
+	new /obj/item/weapon/watertank/janitor/operator(src)
+	new /obj/item/clothing/suit/space/hardsuit/syndi/elite(src)
```
Comment (duncathan) [inline code/game/objects/items/weapons/storage/backpack.dm]: Does it have fire protection?

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 4]
```diff
@@ -488,3 +488,20 @@
...
+	new /obj/item/weapon/gun/projectile/automatic/pistol/APS(src)
+	new /obj/item/ammo_box/magazine/pistolm9mm(src)
+	new /obj/item/ammo_box/magazine/pistolm9mm(src)
+	new /obj/item/weapon/reagent_containers/food/drinks/bottle/vodka/badminka(src)
```
Comment (duncathan) [inline code/game/objects/items/weapons/storage/backpack.dm]: What does badmin vodka do exactly?

Comment (Shapsy) [inline code/game/objects/items/weapons/storage/backpack.dm]: I think it's just fancy normal vodka

File: code/game/objects/items/weapons/tanks/watertank.dm [hunk 5]
```diff
@@ -183,6 +183,40 @@
...
+
+/obj/item/weapon/watertank/janitor/operator/New()
+	..()
+	reagents.add_reagent("mutagen",350)
+	reagents.add_reagent("napalm",125)
+	reagents.add_reagent("welding_fuel",125)
+	reagents.add_reagent("clf3",300)
+	reagents.add_reagent("cryptobiolin",350)
+	reagents.add_reagent("plasma",250)
+	reagents.add_reagent("condensedcapsaicin",500)
```
Comment (duncathan) [inline code/game/objects/items/weapons/tanks/watertank.dm]: Holy shit this is very burn
Nice

**Comment by duncathan** [comment]
Why no flamethrower?
:+1: to the idea for sure

**Comment by LanCartwright** [comment]
People generally avoid the flamethrower. I think it's because of the risk of getting your friends on fire, and that it's so easily made that people don't feel like spending 5 tc on it (even though that 5 tc suppresses crowds and ruins hallways better than anything else).

File: code/game/objects/items/weapons/tanks/watertank.dm (hunk 5 above)
Comment (LanCartwright) [inline code/game/objects/items/weapons/tanks/watertank.dm]: I have this bad feeling that people will start metagaming fire extinguishers. Welding fuel + fire is bad enough already. I can't imagine napalm.

File: code/game/objects/items/weapons/storage/backpack.dm (hunk 3 above)
Comment (LanCartwright) [inline code/game/objects/items/weapons/storage/backpack.dm]: https://github.com/tgstation/-tg-station/blob/c77a9f6cb52a151f356bc06a42b6d907b2f5f554/code/modules/clothing/spacesuits/hardsuit.dm#L271
max_heat_protection_temperature = FIRE_IMMUNITY_SUIT_MAX_TEMP_PROTECT

yup

File: code/game/objects/items/weapons/tanks/watertank.dm [hunk 6]
```diff
@@ -430,4 +436,37 @@
...
+/obj/item/weapon/watertank/janitor/operator
+	name = "backpack water tank"
+	desc = "A New Russian backpack spray for systematic cleansing of carbon lifeforms."
+	icon_state = "waterbackpackjani"
+	item_state = "waterbackpackjani"
+	w_class = 2
+
+/obj/item/weapon/watertank/janitor/operator/New()
+	..()
+	reagents.add_reagent("mutagen",350)
```
Comment (KorPhaeron) [inline code/game/objects/items/weapons/tanks/watertank.dm]: ?

Comment (LanCartwright) [inline code/game/objects/items/weapons/tanks/watertank.dm]: Mutagen is a must have for any spray based chem weapon.

**Comment by TheNightingale** [comment]
Now this is why I added the elite hardsuit. When you fire the sprayer, how far does it spray, and does the plasma ignite?

**Comment by LanCartwright** [comment]
I believe it's 3x4 or 1x4. Goofball changed the backpack sprays. I'm fairly sure it's 1x4.

The plasma sprayed will turn into a gas an the clf3 will ignite the plasma, as will the victim who will be ignited by clf3, stacked in with napalm and welding fuel.

**Comment by LanCartwright** [comment]
whoops.

**Comment by LanCartwright** [comment]
thanks to xxalpha for helping me undo 65 commis and 147 files

**Comment by Supermichael777** [comment]
check to make sure your shit is above the minimum chem threshold(one unit i think) otherwise nothing happens.remember the formulat is (reagent/total reagents)*units sprayed=dose and that must be above the minimum.

**Comment by LanCartwright** [comment]
Testing complete.
Some bugs I overlooked tested and finished.

Getting sprayed without protective equipment is extremely deadly. This is, however easily countered by a fire extinguisher.

It's less versatile than the chemspray, but it is a very safe weapon, and the equipment that it comes with makes it safe for the purchaser.

File: code/modules/uplink/uplink_item.dm [hunk 7]
```diff
@@ -134,6 +134,12 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	item = /obj/item/weapon/storage/backpack/dufflebag/syndie/ammo/fireteam
 	cost = 24 // normally ??
 
+/datum/uplink_item/nukeoffer/firestarter
+	name = "Spetsnaz Pyro bundle"
+	desc = "For systematic suppression of carbon lifeforms in close range: Contains a specialist Pyrotechnic equipment, foreign pistol, two magazines, a pipebomb, and a stimulant syringe."
+	item = /obj/item/weapon/storage/backpack/dufflebag/syndie/firestarter
+	cost = 20
```
Comment (tkdrg) [inline code/modules/uplink/uplink_item.dm]: way too cheap even for the killpack itself, let alone the free suit + minibomb
double the price at the very least

Comment (LanCartwright) [inline code/modules/uplink/uplink_item.dm]: Alright.

**Comment by KorPhaeron** [comment]
@LanCartwright 

Are you going to finish this?

**Comment by LanCartwright** [comment]
Price changed. WIP can be removed. it's finished. avaliable for testing/review.

**Comment by duncathan** [comment]
@LanCartwright

File: code/modules/clothing/under/syndicate.dm [hunk 8]
```diff
@@ -29,3 +30,11 @@
 	icon_state = "camogreen"
 	item_state = "g_suit"
 	item_color = "camogreen"
+/obj/item/clothing/under/syndicate/soviet
```
Comment (KorPhaeron) [inline code/modules/clothing/under/syndicate.dm]: Line spacing please

**Comment by KorPhaeron** [comment]
@LanCartwright 

This looks fine balance/pricing wise, I will merge it as soon as you fix the conflicts

**Pull request closed without merging**